index.html

```diff
@@ -5931,7 +5931,8 @@ <h5>Note regarding the ARIA 1.1 <code>none</code> role.</h5>
 		<div class="role" id="note">
 			<rdef>note</rdef>
 			<div class="role-description">
-				<p>A section whose content is parenthetic or ancillary to the main content of the resource.</p>
+				<p>A <rref>section</rref> whose content qualifies, explains, annotates or is subordinate to some other piece of content</p>
```

Comment:
I was going to say we don't need to reference `section` here, having just looked through other definitions that don't reference `section`... but there is quite a bit of inconsistency.  E.g,  there are roles that do explicitly refer to themselves as `section` (`listitem`), those that define themselves as "sections" of content (`blockquote`, `article`, `code`, etc.)  but do not reference the abstract role, and then even those like `math` which instead just state "content that represents...", bypassing the term section all together.

I don't have a strong preference one way or the other, but calling this out because I do have a preference we make the spec consistent (in another PR, but the decision could impact the linking/wording for this one).

Reply:
That's fair. I wondered about it too, and just copied the first pattern I found. 
FWIW the inheritance relationship is mentioned in the table of characteristics. Maybe that's enough.

For clarity, I think that if the word 'section' refers to the abstract ARIA role (or the HTML element), it should get the appropriate markup sugar. In every other case, we should try to avoid it, and use an appropriate synonym.